primitives/src/locktime/relative.rs

```diff
@@ -39,15 +40,15 @@ pub use units::locktime::relative::*;
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum LockTime {
     /// A block height lock time value.
-    Blocks(Height),
-    /// A 512 second time interval value.
-    Time(Time),
+    Blocks(HeightSpan),
+    /// Intervals of 512 seconds.
+    Time(TimeSpan),
```

Comment:
Changed the name of the u16 newtype wrappers to better represent it function.

Reply:
I'm not convinced that this change assists clarity. In this library we have, for better or for worse, optimized for usage of one level of path to remove ambiguity, so a `relative::Height` is very clear but  a `relative:HeightSpan` is no better. Also this change would require going over all the docs in the module with a fine tooth comb to check them, something I don't really want to do (because it introduces a new term "span"). Also "height span" is never used in the bip, I do see that timespan is though.

Reply:
In Simplicity we use the term "distance" for this (to distinguish "interval" for time). Unfortunately neither English nor Bitcoin jargon has a unique word for this term.

So agreed we should just stick with `Height` and `Time` to refer to both quantities and intervals. The `relative::` and `absolute::` prefixes help.

Reply:
>So agreed we should just stick with Height and Time to refer to both quantities and intervals. The relative:: and absolute:: prefixes help.

Actually not. You are still thinking about them, absolute and relative, as similar. 

the actual `relative::Height` and `absolute::Height` have, theoretically, the same type and nature of values. but it usage its totally different.
The actual `relative::Time` isnt even a unix timestamp. 

my only concern about the naming(you guys have the priority to choose since youre the natives in english) is that it differs the kind and usage. 

I thought about span because in its translation to portuguese is "periodo" which, for me, sounded short and perfect.

>Also this change would require going over all the docs in the module with a fine tooth comb to check them, 

I was using a zed feature for that, and this editor actually is on a pretty beta phase(maybe thats why i couldnt manage to corretly apply the fmt), but its replace feature it far from good since the editor has rust development as its native ambience.

Reply:
They are quite similar conceptually, and their usage is very similar as well. In Miniscript they are exposed using the words "older" and "after" and in many cases these are basically interchangeable.

The set of allowable ranges is very different, as is the encoding, and of course they *are* conceptually different and should not be confused with each other. Hence having separate types.

But the English words "time" and "height" are used for both, and trying to change out words will add to confusion and put us out of step with the ecosystem.